### ISSUE TITLE
xception pretrained models

### ISSUE THREAD

**Issue by spytensor** [YudeWang/deeplabv3plus-pytorch#18]
Hello, the model in http://data.lip6.fr/cadene/pretrainedmodels/xception-b5690688.pth cannot be used? I got the error :size mismatch for conv3.pointwise.weight

**Comment by YudeWang** [comment]
@spytensor 
Please use the pretrained model provided in README. The xception model in this repo is different from official xception structure.

**Comment by spytensor** [comment]
@YudeWang Thanks for your reply, I have solved that problem. By the way, If you have the plan to add demo.py to show result on each image?

**Comment by YudeWang** [comment]
@spytensor 
test.py will save predicted results. You can find the path and check or write another demo.py like that.

**Comment by spytensor** [comment]
@YudeWang Thanks .

**Comment by lxtGH** [comment]
@YudeWang Hi ! What is the mean and vars for using your xception model to train the net???

**Comment by YudeWang** [comment]
@lxtGH I just normalize to [0,1] by /255
